packages/itwinui-css/src/button/base.scss

```diff
@@ -41,6 +41,7 @@ $disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-
 @mixin iui-button-icon {
   display: inline-flex;
   flex-shrink: 0;
+  color: var(--_iui-button-icon-color, currentColor);
```

Comment:
If we are setting `color`, then I think the existing svg `fill` implementation should be changed to use `fill: currentColor` in all places.

Reply:
[3616eda](https://github.com/iTwin/iTwinUI/pull/2554/commits/3616eda9978a5194500fa80b62df740aa253158d)
I've not removed `fill` from `iui-field`, since it is used in `iui-select`.

Reply:
This increases the scope (and potential introduction of issues) significantly. I've strictly focused on changes needed to address https://github.com/iTwin/appui/issues/1305